pcap/net_parse.go

```diff
@@ -48,18 +48,27 @@ func (fact *tcpStreamFactory) New(netFlow, tcpFlow gopacket.Flow, _ *layers.TCP,
 	return newTCPStream(fact.clock, netFlow, fact.outChan, fact.fs)
 }
 
+type NetworkTrafficObserver func(gopacket.Packet)
+
 type NetworkTrafficParser struct {
 	pcap  pcapWrapper
 	clock clockWrapper
+
+	observer NetworkTrafficObserver
```

Comment:
Please document the purpose of this field.

Reply:
Fixed.